src/pal/tools/gen-buildsys-gcc.sh

```diff
@@ -45,15 +45,21 @@ else
     exit 1
 fi
 
-if [ -z "$CC" ]; then
+if [ -z "$DOTNET_CC" ]; then
```

Comment:
Could you please rather use `CLR_`  prefix in the name, similar to what we use for CMake variables passed as parameters?

Reply:
Sure, I first thought about `CLR_` prefix, then realized that these scripts could be shared with CoreFX and CoreSetup (maybe via arcade repo). If that is a right goal, would `DOTNET_` makes more sense or is `CLR_` considered as neutral prefix across the repositories?

Reply:
We use `CLR_` prefix in corefx / core-setup cmake variables too.

Reply:
Updated with `CLR_` prefix.